### PR TITLE
Issue 50 - Use Video.js CDN instead of keep old version

### LINKED ISSUE #50: Take Video.js out of this package
We should create collective.js.videojs to make maintenance of the package easier.

Code repo is here: https://github.com/videojs/video.js

We may probably use the CDN hosted version of the package.

### PR DESCRIPTION
**Pull request by rodfersou**
closes #50

### CODE REVIEW

File: src/sc/embedder/content/templates/embeddervideojs.pt [hunk 1]
```diff
@@ -30,6 +30,7 @@
     <tal:notajax tal:condition="python:not ajax_load or ajax_include_head">
         <div tal:replace="structure provider:plone.htmlhead" />
         <link tal:replace="structure provider:plone.htmlhead.links" />
+        <link href="//vjs.zencdn.net/5.8/video-js.min.css" rel="stylesheet">
```
Comment (cleberjsantos) [inline src/sc/embedder/content/templates/embeddervideojs.pt]: @hvelarde @rodfersou There'll be interesting to implement here?

https://blog.cloudflare.com/an-introduction-to-javascript-based-ddos/
https://hacks.mozilla.org/2015/09/subresource-integrity-in-firefox-43/

**Comment by hvelarde** [comment]
@cleberjsantos awesome! do you want to work on that on a new branch?

**Comment by cleberjsantos** [comment]
:+1: Yes ;)

**Comment by hvelarde** [comment]
@cleberjsantos I added #55 feel free to work on branch `issue_55`.

**Comment by cleberjsantos** [comment]
@hvelarde Nice, tks.

**Comment by rodfersou** [comment]
nice, we should use it here too https://github.com/simplesconsultoria/collective.texttospeech/blob/master/src/collective/texttospeech/profiles/default/jsregistry.xml#L3

**Comment by rodfersou** [comment]
I just don't know if these new attributes of ``<script>`` tag would leave some accessibility issues to us..

**Comment by cleberjsantos** [comment]
@rodfersou  @hvelarde  Hmm, then let's test :) but I believe that not we'll have problems, see w3c (W3C Candidate Recommendation 13 November 2015) https://w3c.github.io/webappsec-subresource-integrity/

**Comment by hvelarde** [comment]
I don't see any issue with accessibility; if one arises we will fix it later.

**Pull request merged**